internal/install/license_key_fetcher.go

```diff
@@ -0,0 +1,65 @@
+package install
+
+import (
+	"context"
+	"strconv"
+
+	log "github.com/sirupsen/logrus"
+
+	"github.com/newrelic/newrelic-cli/internal/credentials"
+	"github.com/newrelic/newrelic-cli/internal/install/recipes"
+)
+
+// relies on the Nerdgraph service
+type ServiceLicenseKeyFetcher struct {
+	client recipes.NerdGraphClient
+}
+
+type LicenseKeyFetcher interface {
+	FetchLicenseKey(context.Context) (string, error)
+}
+
+func NewServiceLicenseKeyFetcher(client recipes.NerdGraphClient) LicenseKeyFetcher {
+	f := ServiceLicenseKeyFetcher{
+		client: client,
+	}
+
+	return &f
+}
+
+func (f *ServiceLicenseKeyFetcher) FetchLicenseKey(ctx context.Context) (string, error) {
```

Comment:
This code is mostly a duplicate of this init code that is run the first time a CLI command is executed after installation:
https://github.com/newrelic/newrelic-cli/blob/master/cmd/newrelic/command.go#L142-L167 

Could be an opportunity to DRY up the code.